src/Criteo.OpenApi.Comparator.Cli/Program.cs

```diff
@@ -45,15 +47,41 @@ public static int Main(string[] args)
         }
 
         private static bool TryReadFile(string path, out string fileContent)
+        {
+            bool readOk = TryReadDistantFile(path, out fileContent);
+            if (!readOk)
+            {
+                TryReadLocalFile(path, out fileContent);
+            }
+            return readOk;
+        }
+
+        private static bool TryReadDistantFile(string url, out string fileContent)
+        {
+            try
+            {
+                using HttpClient wc = new HttpClient();
+                fileContent = wc.GetStringAsync(url).Result;
```

Comment:
Consider using `async/await` for the `HttpClient.GetStringAsync(url)` call to avoid blocking the thread. This can improve the performance of your application. [important]